README.md

```diff
@@ -719,7 +719,7 @@ doas pkg_add gmake libexecinfo llvm pcre2
 This will build ponyc and compile helloworld:
 
 ```bash
-gmake verbose=true default_pic=true bits=64
+gmake verbose=true bits=64
```

Comment:
This is still missing `default_pic=true`, right?

Reply:
AAUI, this pull request moves default_pic=true from the command-line to Makefile-ponyc.